### DISCUSSION TITLE
AntiPopup Compatibility

### DISCUSSION THREAD

**Discussion by R00tB33rMan** [QuickShop-Community/QuickShop-Hikari#1555] [Ideas]
Currently, AntiPopup + QuickShop effectively makes muted users unable to register a price. Seemingly, a specific hook or check needs to be implemented so that this is no longer an issue or concern. I've tried all chat-related settings through QuickShop, such as:

```
# Force bukkit chat handler (https://github.com/KaiKikuchi/QuickShop/issues/10)
force-bukkit-chat-handler: false
and
# Ignore cancelled chat event, can sometimes improve compatibility with some chat plugins.
ignore-cancel-chat-event: true (both true/false) to no avail.
  
Link to AntiPopup: https://github.com/KaspianDev/AntiPopup

**Comment by YuanYuanOwO** [comment]
can you try `/quickshop price` ?

**Comment by Ghost-chu** [comment]
If you're running a QuickShop server on BungeeCord/Velocity network, consider install their compatibility module to take the chat handler from Spigot server to Proxy.




https://ghost-chu.github.io/QuickShop-Hikari-Documents/docs/compat-modules/bungeecord
https://ghost-chu.github.io/QuickShop-Hikari-Documents/docs/compat-modules/velocity
&nbsp;


Ghost_chu
***@***.***



&nbsp;




------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;; 
Date: 2024年5月12日(星期天) 晚上11:02
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; 
Subject: [Ghost-chu/QuickShop-Hikari] AntiPopup Compatibility (Discussion #1555)














 
Currently, AntiPopup + QuickShop effectively makes muted users unable to register a price. Seemingly, a specific hook or check needs to be implemented so that this is no longer an issue or concern. I've tried all chat-related settings through QuickShop, such as:
 force-bukkit-chat-handler: false``` and ```  # Ignore cancelled chat event, can sometimes improve compatibility with some chat plugins.   ignore-cancel-chat-event: true``` (both true/false) to no avail.      Link to AntiPopup: https://github.com/KaspianDev/AntiPopup

**Reply by R00tB33rMan** [reply]
Thanks to everyone who replied! I'll give this a shot. I didn't think this was relevant, but clearly, it must be.
